src/views/Login.js

```diff
@@ -42,12 +42,13 @@ export class Login extends Component {
       );
       this.props.setLoadingStatus(false);
 
-      const { isLoggedIn } = this.props.userData;
-      isLoggedIn
-        ? this.props.history.push('/home')
-        : this.props.handleError(
-            'Incorrect email or password. Please try again.'
-          );
+      const { isLoggedIn } = this.props.userData;   
+      if(isLoggedIn === false) 
+      return this.props.handleError('Incorrect email or password. Please try again.')
+      const isSuperAdmin = (this.props.user.email === process.env.adminEmail);
```

Comment:
So do mean that the system should have only one admin?

Reply:
No really, however I can't catch the user role while login

Reply:
So you should make sure that in the token you have a role or after logging, you can find a way of knowing the role of that user.